packages/common/src/ui/layout/tables/columns/EditableQuantityColumn.tsx

```diff
@@ -70,5 +70,5 @@ export const getEditableQuantityColumn = <
     );
   },
   label: 'label.pack-quantity',
-  accessor: (row: T) => String(row.quantity),
+  accessor: ({ rowData }) => String(rowData.quantity),
```

Comment:
a bunch of lines changing `(row) => row.X` or `(rowData) => rowData.X` to `({rowData}) => rowData.X`